tutorials/text/mathtext.py

```diff
@@ -94,7 +94,7 @@
 
 .. math::
 
-    \frac{3}{4} \binom{3}{4} \genfrac{}{}{0}{}{3}{4}
+    \frac{3}{4} \binom{3}{4} \genfrac{}{}{0pt}{}{3}{4}
```

Comment:
This dimension appears to be required by LaTeX, but not our parser. I couldn't find anything in the parser that could require it, (if we wanted to do that)?

cc @anntzer

Reply:
Indeed, `\genfrac` behaves differently between mathtext and TeX: TeX wants (quite reasonably) a unit whereas we have not implemented support for it.  I don't actually know what's the unit we implicitly use?
I guess adding support for units there is not "too" hard (mostly just figuring out the right multiplier); but in the meantime I guess you could just exclude that from the pdf build using `.. only::` or `.. ifconfig::`?

Reply:
Well this is in `.. math::`; according to the build here, it works fine with the unit in regular HTML builds, too. It's just inconsistent with the Matplotlib version in the plain text source.

Reply:
I guess you should mostly just open an issue for supporting `{0pt}` on our side?  (which seems like a reasonable request)  Or did I miss something?

Reply:
Yes, as it seems I didn't miss anything, #19001.